### ISSUE TITLE
megassh doc

### ISSUE THREAD

**Issue by pfuntner** [pfuntner/toys#100]
`megassh` is one of my favorite tools and I need to do a doc page for it.

I had a new use case today that I want to be sure that I include:

### Copy a remote tree using `tar`
```
$ megassh -qe target -- 'cd target-root; tar -czf - target-dir | base64' | base64 -d | tar -vxzf -
```

I had to tweak `megassh` to get this to work but it did work as desired!

**Comment by pfuntner** [comment]
I've done a lot of work in the [megassh-doc branch](https://github.com/pfuntner/toys/tree/megassh-doc) but don't have an ssh target which to develop decent examples.

**Comment by pfuntner** [comment]
I finished the doc a merged the branch with #102
